lab-anthony/lib/greet.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+function greet(name){
+  const person = name || '...not you.';
+  console.log('Hello ' + process.argv[2]);
+  return 'Hello ' + person;
+}
+greet();
+
+module.exports.greet = greet;
```

Comment:
This code works! And we have not gone over this in class very much, but it is breaking a nodejs convention. 

when exporting a single value use
`module.exports = value`
when exporting a value on the exports object use
`exports.key = value`

so you wont see 
`module.exports.key = value`